homeassistant/components/iskra/strings.json

```diff
@@ -0,0 +1,80 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Configure Iskra Device",
+        "description": "Enter the IP address of your Iskra Device and select protocol.",
+        "data": {
+          "host": "Host"
+        }
+      },
+      "authentication": {
+        "title": "Configure Rest API Credentials",
+        "description": "Enter username and password",
+        "data": {
+          "username": "Username",
+          "password": "Password"
+        }
+      },
+      "modbus_tcp": {
+        "title": "Configure Modbus TCP",
+        "description": "Enter Modbus TCP port and device's Modbus address.",
+        "data": {
+          "port": "Port",
+          "address": "Modbus address"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "Failed to connect to the device",
+      "invalid_auth": "Invalid authentication",
+      "unknown": "Unexpected error"
+    },
+    "abort": {
+      "already_configured": "Device is already configured"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "total_active_power": {
+        "name": "Total active power"
+      },
+      "total_apparent_power": {
+        "name": "Total apparent power"
+      },
+      "total_reactive_power": {
+        "name": "Total reactive power"
+      },
+      "phase1_power": {
+        "name": "Phase 1 power"
+      },
+      "phase2_power": {
+        "name": "Phase 2 power"
+      },
+      "phase3_power": {
+        "name": "Phase 3 power"
+      },
+      "phase1_voltage": {
+        "name": "Phase 1 voltage"
+      },
+      "phase2_voltage": {
+        "name": "Phase 2 voltage"
+      },
+      "phase3_voltage": {
+        "name": "Phase 3 voltage"
+      },
+      "phase1_current": {
+        "name": "Phase 1 current"
+      },
+      "phase2_current": {
+        "name": "Phase 2 current"
+      },
+      "phase3_current": {
+        "name": "Phase 3 current"
+      },
+      "frequency": {
+        "name": "Frequency"
```

Comment:
```suggestion
```
Can be removed since it will use device class translations

Reply:
commited suggestion